packages/api/src/platforms/vtex/clients/search/index.ts

```diff
@@ -97,6 +98,9 @@ export const IntelligentSearch = (
   ): Promise<SuggestedTerms> =>
     fetchAPI(`${base}/api/split/suggestion_search/?q=${args.query}`)
 
+  const topSearches = (): Promise<{ searches: Search[] }> =>
```

Comment:
You need to rebase your branch with the main, in order to get the new type for that
```suggestion
  const topSearches = (): Promise<Suggestion> =>
```